### DISCUSSION TITLE
Multiple package install/upgrade with oneliner

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-cli#1911] [Q&A] [answered]
All other package managers allow( one way or another) installing multiple packages with single command e.g. `dnf install app1 app2`, `apt install app1 app2`, `brew install $(package.txt)` or something similar. 

is there anyway to do that with the winget like `winget install list.txt` or `winget install apps.config` or with a xml instead of `winget install app1; winget install app2`

**Comment by denelon** [comment, marked as answer]
Yes, take a look at `winget export -?` and `winget import -?`.

**Reply by kdpuvvadi** [reply]
huh! i was aware of this. thanks mate 👍
